apps/wallet/src/ui/app/components/explorer-link/Explorer.ts

```diff
@@ -23,15 +23,17 @@ function getExplorerUrl(
     const base =
         apiEnv === API_ENV.local
```

Comment:
Not related with the changes but probably would be better to always use the prod explorer. I don't think that people using the local network always run the explorer as well

Reply:
I thought that was the use for the custom RPC field.

Reply:
Reply